src/tsconfig.no-any.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "checkJs": true,
+    "lib": ["es2020", "dom", "dom.iterable"],
+    "jsx": "react-jsx",
+    "jsxImportSource": "preact",
+    "module": "es2020",
+    "moduleResolution": "node",
+    "noEmit": true,
+    "strict": true,
+    "target": "ES2020",
+
+    // Let argument to catch statement be `any` rather than `unknown`.
```

Comment:
Comments here in JSON...is that allowable?

Reply:
TypeScript is one of several tools that accepts a superset of JSON in their config files. See https://github.com/microsoft/TypeScript/issues/4987.